src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+
+html {
+  --active-color: #00acdc;
+}
+
+body {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo__container {
+  display: flex;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  list-style: none;
+  display: flex;
+  padding: unset;
+  box-sizing: border-box;
+}
+
+.nav__item {
+  margin: 0;
+  display: inline-block;
+  bottom: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:link {
+  position: relative;
+  display: block;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  line-height: 60px;
+  height: 60px;
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.is-active:link {
```

you can merge these styles
```suggestion
.nav__link:hover,
.is-active:link {
```